protocol/v1/qbft/instance/msgcont/inmem/inmem_test.go

```diff
@@ -163,18 +163,7 @@ func TestMessagesContainer_AddMessage(t *testing.T) {
 			Data:       []byte{1, 1, 1, 1},
 		},
 		Signature: nil,
-		Signers:   []spectypes.OperatorID{4, 5},
-	}, nil)
-	require.Len(t, c.ReadOnlyMessagesByRound(1), 1)
-	require.Len(t, c.ReadOnlyMessagesByRound(2), 0)
-	c.AddMessage(&specqbft.SignedMessage{
```

Comment:
why deleting this action?

Reply:
@nivBlox Because both signers `[]spectypes.OperatorID{4, 5}` and `[]spectypes.OperatorID{4}` aren't considered duplicates in spec, so I replaced these two cases with one case with signers `[]spectypes.OperatorID{1, 2, 3, 4}`.